### ISSUE TITLE
FX not affected by region feature of the V12

### ISSUE THREAD

**Issue by Dolgrenn** [ghost-fvtt/fxmaster#726]
### Expected Behavior

FX should be blocked by the region I created

### Current Behavior

the FX still appear in the region I created to block them

### Steps to Reproduce

1. Create a region using the tool integrated into v12
2. activate the region option blocking FX
3. activate an effect on the current scene

### Version

v4.0.2

### Foundry VTT Version

v12

### Operating System

windows

### Browser / App

Other

### Game System

Vaesen

**Comment by ghost91-** [comment]
V12 is not supported yet, see https://github.com/ghost-fvtt/fxmaster/issues/718

**Comment by Aioros** [comment]
Assuming that everything goes well with the v12 update, I think that this is going to be the most requested feature. An easy start would be to treat "Suppress Weather" regions just like mask drawings, and that doesn't seem too difficult (I might have a PR basically ready), but I wonder if there's any interest in creating a custom region type for that. The behavior of the region could even define the specific particle effect for that region, if masking individual particle effects is still a desired feature.

**Comment by ghost91-** [comment]
@Aioros I agree, this is likely a desired feature and should eventually just replace the drawing mask functionality all together. As a first step, just supporting the "Suppress Weather" regions sounds like a decent idea. Unfortunately, I currently don't have the capacity to do that. Feel free to create a PR, though.

**Comment by Nedrapter** [comment]
Experiencing this as well, I noticed that if the effect is added from the Scene's configuration, it does get supressed. However this way you can't add multiple effects and it's also not convenient.
